deb-get

```diff
@@ -2413,6 +2413,20 @@ function deb_mullvad-vpn() {
     SUMMARY="VPN Service based in Sweden that does not collect user data."
 }
 
+function deb_xournalpp() {
+    CODENAMES_SUPPORTED="bionic bullseye focal"
+    get_github_releases "https://api.github.com/repos/xournalpp/xournalpp/releases/latest"
+    if [ "${ACTION}" != "prettylist" ]; then
+        if [[ "${UPSTREAM_CODENAME}" == *"${CODENAMES_SUPPORTED}"* ]]; then
```

Comment:
There is no need for this check, as it is already done before this function is called.